build/shared/examples/04.Communication/SerialPassthrough/SerialPassthrough.ino

```diff
@@ -22,12 +22,24 @@
   by Erik Nyquist
 */
 
+uint32_t currentBaud = 9600;
+
 void setup() {
-  Serial.begin(9600);
-  Serial1.begin(9600);
+  while (!Serial) {
+    ; // wait for serial port to connect. Needed for native USB
+  }
+  currentBaud = Serial.baud();
```

Comment:
I wonder if we also added the `baud()` method to HardwareSerial? If not, this would make the example tied to CDC serial and unusable on HardwareSerial (or any other platforms that do not offer the baud() method). I wonder if there is a way to detect that `baud()` is or is not available...